Gopkg.toml

```diff
@@ -198,3 +193,7 @@ required = [
 [[constraint]]
   name = "github.com/sensu/yaml"
   revision = "704b36ea12f40d32ffe6bd109c2ddc894af28dde"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/robertkrimen/otto"
```

Comment:
No specific version or revision constraint for this new dependency?
I'm not a fan of "YOLO pulling master" :)

Reply:
The project hasn't made any releases. I think in this case it's better to use master and commit it to vendor instead of pinning a specific SHA. If we notice some bugs we can try to upgrade or fork.